Samples/ThreeStepAuthMVC/Program.cs

```diff
@@ -0,0 +1,38 @@
+using Celigo.ServiceManager.NetSuite.TSA;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddControllersWithViews();
+builder.Services.AddScoped<IRequestTokenService,RequestTokenService>();
+builder.Services.AddScoped<IAccessTokenService,AccessTokenService>();
+builder.Services.AddScoped<IRevokeTokenService,RevokeTokenService>();
+builder.Services
+    .AddOptions()
+
+    .AddHttpClient()                
+
+    .AddSingleton<ITokenService, DefaultTokenService>();
```

Comment:
These are Obsolete. Instead, lets use `IAccessTokenService` and `IRequestTokenService`

Reply:
updated as mentioned